source/features/dim-bots.tsx

```diff
@@ -53,8 +53,17 @@ function undimBots(event: DelegateEvent): void {
 function init(signal: AbortSignal): void {
 	for (const bot of $$(commitSelectors)) {
 		// Exclude co-authored commits
-		if ($$('a', bot.parentElement!).every(link => link.matches(commitSelectors))) {
-			bot.closest('.commit, .Box-row')!.classList.add(dimBots.class);
+		if ($$('a', bot.parentElement!).every(link => {
+			const href = link.getAttribute('href');
+			// If href starts with /apps/ it's a bot
+			return href?.startsWith('/apps/') ? true : link.matches(commitSelectors);
```

Comment:
I think why I write it because of the browser caches. hahaha, I write it in the `commitSelectors` first but it doesn't work. So I change a way to do it. Now I update it.